Tests/TensorFlowTests/TensorAutoDiffTests.swift

```diff
@@ -411,6 +416,7 @@ final class TensorAutoDiffTests: XCTestCase {
         ("testConcatenationPlusPlus", testConcatenationPlusPlus),
         ("testConcatenated", testConcatenated),
         ("testTransposed", testTransposed),
+        ("testSigmoid", testSigmoid),
```

Comment:
The definition of `testSigmoid()` is placed between `testRelu()` and `testSoftmax()`. This line should be moved below `("testRelu", testRelu)` for consistency.

Reply:
Moved `testSigmoid` below `testRelu` line.

Reply:
Sorry to be nitpicky about this but it'd be better to move the `testSigmoid()` definition above the `testRelu()` definition instead so it matches the order in which they are defined in `Math.swift`.

Reply:
It makes sense.
Moved `testRelu` definition.
(In first commit, I was puzzled which should be appear first `testRelu` or `testSigmoid`. And somehow left them inconsistent order😅)

Reply:
Thanks @t-ae.

Reply:
Thanks @t-ae. :)